src/stories/components/Teaser/Teaser.stories.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Teaser } from './Teaser';
+
+/**
+ * Example Component: Teaser
+ */
+export default {
+    title: 'Example/Teaser',
+    component: Teaser,
+};
+
+const Template = (args) => <Teaser {...args} />;
+
+// Dark line teaser
+export const Dark = Template.bind({});
+Dark.args = {
+    modifierClasses: '',
+    url: '#',
+    title: 'Title with a Dark Teal Line',
+    text: 'Dark Teal Line',
+};
+
+// Light line teaser
+export const Light = Template.bind({});
+Light.args = {
+    modifierClasses: 'teaser--light',
+    url: '#',
+    title: 'Title with a Teal Line',
+    text: 'Teal line',
+};
+
+// Disabled teaser
+export const Disabled = Template.bind({});
+Disabled.args = {
+    modifierClasses: 'teaser--disabled',
+    url: '#',
```

Comment:
The disabled teaser should probably not link somewhere. Maybe disable the link and keep the url to pass in?

Reply:
That is a very good observation.

Reply:
I disabled the link redirection in the styles.scss (see line 49)